cirq-core/cirq/ops/common_gates.py

```diff
@@ -92,6 +92,30 @@ class XPowGate(eigen_gate.EigenGate):
     def __init__(
         self, *, exponent: value.TParamVal = 1.0, global_shift: float = 0.0, dimension: int = 2
     ):
+        """Initialize an XPowGate.
+
+        Args:
+            exponent: The t in gate**t. Determines how much the eigenvalues of
+                the gate are scaled by. For example, eigenvectors phased by -1
```

Comment:
s/scaled/phased/

The word "scaled" suggests a possible change to absolute value (which does not change - all eigenvalues of a unitary gate have unit absolute value).

(Moot if you choose to follow the suggestion below.)

Reply:
Done, although this phrasing is copied from EigenGate. I'll update it there as well.